MaterialSkinExample/MainForm.cs

```diff
@@ -82,9 +83,9 @@ private void updateColor()
             {
                 case 0:
                     materialSkinManager.ColorScheme = new ColorScheme(
-                        materialSkinManager.Theme == MaterialSkinManager.Themes.DARK ? Primary.Teal500 : Primary.Indigo500,
-                        materialSkinManager.Theme == MaterialSkinManager.Themes.DARK ? Primary.Teal700 : Primary.Indigo700,
-                        materialSkinManager.Theme == MaterialSkinManager.Themes.DARK ? Primary.Teal200 : Primary.Indigo100,
+                        !materialSkinManager.EnabledLightTheme ? Primary.Teal500 : Primary.Indigo500,
+						!materialSkinManager.EnabledLightTheme ? Primary.Teal700 : Primary.Indigo700,

```

Comment:
This is also a breaking change, we should come up with some way of keeping the check `materialSkinManager.Theme == MaterialSkinManager.Themes.DARK` valid, maybe create a compatible static class?